### ISSUE TITLE
Acknowledge contributors

### ISSUE THREAD

**Issue by vlsi** [javacc/javacc-8-java#2]
javacc-8-java contains code contributed by several people, however its git history is blank: https://github.com/javacc/javacc-8-java/graphs/contributors

For instance, I contributed this code: https://github.com/javacc/javacc-8-java/blob/0942ada4a5ebdf38a510c861217f1bbbf4de4f10/src/main/java/org/javacc/java/ParserCodeGenerator.java#L1838 , and I want that the change to be attributed appropriately.

**Comment by zosrothko** [comment]
Yes, you are rigth in your claim. I will make your contribution appearing if possible and in any case, you will be added as a contributor in the pom.xml
